docker-compose-s3.yml-dist

```diff
@@ -0,0 +1,18 @@
+db:
+  image: postgres:9.3
+client_build:
+  extends:
+    file: docker-compose-base.yml
```

Comment:
Yeah, I'm hoping we can keep the image for the Django server built in a way that the only dev / prod difference is env vars. Don't think we'll be able to use docker-compose in prod, but we should be able to play with prod-ish features in dev like S3 by just juggling env vars